backend/src/test/java/com/engineering/thesis/backend/BackendApplicationTests.java

```diff
@@ -2,19 +2,12 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.jdbc.Sql;
-import org.springframework.test.context.jdbc.SqlGroup;
-import org.springframework.test.context.jdbc.Sql.ExecutionPhase;
-
-@SqlGroup({
-		@Sql(executionPhase = ExecutionPhase.AFTER_TEST_METHOD ,scripts = "classpath:delete_data_after_test.sql")
```

Comment:
we perform tests without any initial data and checking if controll and service layers work properly.

Reply:
will remove sql scripts in tests resources in refactor task.

Reply:
Just a remainder that this would be nice to keep track of, maybe a // TODO in next changes?